src/nvim/testdir/unix.vim

```diff
@@ -2,14 +2,20 @@
 " Always use "sh", don't use the value of "$SHELL".
 set shell=sh
 
+" Only when the +eval feature is present. 
+if 1
+  " While some tests overwrite $HOME to prevent them from polluting user files,
+  " we need to remember the original value so that we can tell external systems
+  " where to ask about their own user settings.
+  let g:tester_HOME = $HOME
+endif
+
+source setup.vim
+
+" TODO: upstream?
+" Initially added in https://github.com/neovim/neovim/commit/4a5bc6275d090.
+" Appears to make sense as in "adjust for set shell=sh".
 if has('win32')
   set shellcmdflag=-c shellxquote= shellxescape= shellquote=
   let &shellredir = '>%s 2>&1'
-  set shellslash
```

Comment:
Why is this removed? This shouldn't break anything and `shellslash` goes together with `sh` shell.

Reply:
Will remove it completely - according to https://github.com/vim/vim/pull/4749 it should not be necessary (i.e. it should follow "set shell=sh").

Reply:
..but nvim does not have that: https://github.com/vim/vim/blob/8c5a278fc508da6dfe50e69b6ee734451aa4eafb/src/option.c#L4173-L4232

Reply:
Will move it to setup.vim